hpx/runtime.hpp

```diff
@@ -355,6 +356,19 @@ namespace hpx
         notification_policy_type::on_startstop_type on_stop_func_;
         notification_policy_type::on_error_type on_error_func_;
     };
+
+    namespace util {
+        ///////////////////////////////////////////////////////////////////////////
+        // retrieve the command line arguments for the current locality
+        HPX_API_EXPORT bool retrieve_commandline_arguments(
+            hpx::program_options::options_description const& app_options,
+            hpx::program_options::variables_map& vm);
+
+        ///////////////////////////////////////////////////////////////////////////
+        // retrieve the command line arguments for the current locality
+        HPX_API_EXPORT bool retrieve_commandline_arguments(
+            std::string const& appname, hpx::program_options::variables_map& vm);
+    }    // namespace util
```

Comment:
Wouldn't these function declarations belong into the new module command_line handling?

Reply:
They've intentionally been moved out of the module because they are free functions that access the runtime and command line handling through the global runtime object. There could be a version of this function that is a member function of `runtime_configuration`, but this specific function belongs close to the runtime.